File: C++/C++Basic/08.for-Loop-Exercise/05.salary.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int countTabs, salary;
    cin >> countTabs >> salary;

    for (int tab = 1; tab <= countTabs; tab++)
    {
        string site;
        cin >> site;

        if (site == "Facebook")
        {
            salary -= 150;
        }
        else if (site == "Instagram")
        {
            salary -= 100;
        }
        else if (site == "Reddit")
        {
            salary -= 50;
        }

        if (salary <= 0)
        {
            cout << "You have lost your salary." << endl;
            break;
        }
    }

    if (salary > 0)
    {
        cout << salary << endl;
    }

    return 0;
}

File: Python/Basic/08.For-Loop-Exersice/05.salary.py
tabs = int(input())
salary = int(input())

for tab in range(1, tabs + 1):
    open_tabs = input()

    if open_tabs == "Facebook":
        salary -= 150
    elif open_tabs == "Instagram":
        salary -= 100
    elif open_tabs == "Reddit":
        salary -= 50
    
    if salary <= 0:
        print(f"You have lost your salary.")
        break
else:
    print(salary)